tdrs-backend/tdpservice/search_indexes/management/commands/clean_and_reparse.py

```diff
@@ -128,11 +137,14 @@ def __delete_records(self, file_ids, new_indices, log_context):
                 raise e
         return total_deleted
 
-    def __delete_errors(self, file_ids, log_context):
+    def _delete_errors(self, file_ids, log_context):
         """Raw delete all ParserErrors for each file ID."""
         try:
             qset = ParserError.objects.filter(file_id__in=file_ids)
+            count = qset.count()
+            logger.info(f"Deleting {count} parser errors.")
             qset._raw_delete(qset.db)
```

Comment:
is `_raw_delete()` fast enough that we don't need a paginator? Does PG have an optimized solution here but not above?

Reply:
The reason we don't need to use pagination on the deletes is because nothing is pulled into main memory. Every operation up to that point was being lazy evaluated. When we tell Django to `_raw_delete` it lets the DB engine execute the query and only returns the number of rows that were deleted. As opposed to returning a `dict` of model items that were deleted.